pelicanconf.py

```diff
@@ -0,0 +1,75 @@
+
+import datetime
+# Basic information about the site.
+SITENAME = 'Apache HTTP Server Project'
+SITEDESC = 'The official website of the Apache HTTP Server Project'
+SITEDOMAIN = 'httpd.apache.org'
+SITEURL = 'https://httpd.apache.org'
+SITELOGO = 'https://httpd.apache.org/images/httpd_logo_wide_new.png'
+SITEREPOSITORY = 'https://github.com/apache/httpd-site/blob/main/content/'
+CURRENTYEAR = datetime.date.today().year
+TRADEMARKS = 'Apache HTTP Server, Apache, and the Apache feather logo are trademarks of The Apache Software Foundation.'
+TIMEZONE = 'UTC'
+# Theme includes templates and possibly static files
+THEME = 'theme/apache'
+# Specify location of plugins, and which to use
+PLUGIN_PATHS = [ '/Users/covener/SRC/infrastructure-actions/pelican/migration/../plugins',  ]
```

Comment:
Is that correct? The path looks like a path on your personal device.

Reply:
ty, will look a this along with some hints from Daniel about testing